tools/release_notes.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint:disable=invalid-name
+
+"""Print changelog of specified version with markdown stripped"""
+
+import sys
+from pathlib import Path
+
+if __name__ == "__main__":
+    cur_version = sys.argv[1]
+
+    with open(Path(__file__).parent.parent / "CHANGELOG.md", encoding="UTF-8") as f:
+        changelog_lines = f.readlines()
+
```

Comment:
nit:

```python
changelog_lines = Path(__file__).joinpath("../../CHANGELOG.md").open(encoding="UTF-8").readlines()
```

Reply:
This doesn't close the file I think, but I can do `with Path(...).joinpath(...).open() as f`!

Reply:
Mh, it needs one `.parent` because otherwise it tries to construct the path `tools/release-notes.py/../../CHANGEOG.md`

Reply:
Oh you can do:

```python
changelog_lines = Path(__file__).joinpath("../../CHANGELOG.md").read_text(encoding="UTF-8").splitlines()
```

Reply:
Ah, cool, didn't know about `splitlines`. Will keep in mind for next time :)